android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -72,32 +73,113 @@
 	private static enum reloadTypes {
 		DEFAULT, DATA, HTML, URL
 	}
-	
+
 	private reloadTypes reloadMethod = reloadTypes.DEFAULT;
 	private Object reloadData = null;
-	
-	private class TiWebView extends WebView
-	{
+
+	private class TiWebView extends WebView {
```

Comment:
Looks like you're not using the same formatting preferences we had, so there are a lot of formatting changes here.

Please use the formatting settings from our standards repo: https://github.com/appcelerator/appc-standards/tree/master/languages/java